client/src/featureform/register.py

```diff
@@ -2878,12 +2878,82 @@ def apply(self):
         else:
             state().create_all(self._stub)
 
+    def run(self):
+        if self._dry_run:
+            print(state().sorted_list())
+            return
+
+        if self.local:
+            state().create_all_local()
+        else:
+            resource_state = state()
+            resource_state.create_all(self._stub)
+            resources = resource_state.sorted_list()
+            while True:
+                # get the status for each resource so we can display it
+                statuses = self.get_statuses(resources)
+                self.display_status(statuses)
```

Comment:
The time.sleep should probably get pulled up into here if you need it, this also needs a break